utils/getUser.js

```diff
@@ -3,8 +3,9 @@ import { getSession } from 'next-auth/client';
 import { GET_USER } from '@/utils/apollo/templates/User';
 import { initializeApollo } from '@/utils/apollo/client';
 
-const getUser = async () => {
-  const userSession = await getSession();
+const getUser = async (req) => {
```

Comment:
oh true. This is only applicable for server side code. Can you do `req = null` so it works without a parameter?

Reply:
or if it works now with the existing code that use getUser then you don't have to change it. I think context/spaces/index.js uses it.

Reply:
getUser without anything passed in still works in context/spaces/index.js but in /dashboard it returns undefined if req isn't passed in